spring-cloud-sleuth-zipkin/src/main/java/org/springframework/cloud/sleuth/zipkin/HttpZipkinSpanReporter.java

```diff
@@ -28,7 +29,19 @@
 	 */
 	public HttpZipkinSpanReporter(RestTemplate restTemplate, String baseUrl, int flushInterval,
 			SpanMetricReporter spanMetricReporter) {
-		this.sender = new RestTemplateSender(restTemplate, baseUrl);
+		this(restTemplate, baseUrl, flushInterval, spanMetricReporter, false);
+	}
+
+	/**
+	 * @param restTemplate {@link RestTemplate} used for sending requests to Zipkin
+	 * @param baseUrl       URL of the zipkin query server instance. Like: http://localhost:9411/
+	 * @param flushInterval in seconds. 0 implies spans are {@link #flush() flushed} externally.
+	 * @param spanMetricReporter service to count number of accepted / dropped spans
+	 * @param thrift use thrift encoding, default is JSON
+	 */
+	public HttpZipkinSpanReporter(RestTemplate restTemplate, String baseUrl, int flushInterval,
+								SpanMetricReporter spanMetricReporter, boolean thrift) {
```

Comment:
there's another json encoding coming soon.. not sure we'll want to use boolean here, especially as a property. Probably a stringification of the enum name is a better choice